src/Futhark/CLI/LSP.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Futhark.CLI.LSP (main) where
+
+import Control.Concurrent.MVar (newMVar)
+import Control.Monad.IO.Class (MonadIO (liftIO))
+import Futhark.LSP.Handlers (handlers)
+import Futhark.LSP.Utils (debug, emptyState)
+import Language.LSP.Server
+import Language.LSP.Types
```

Comment:
While we're normally OK with unqualified imports in the compiler, LSP exports such an enormous amount of names that it may be better to explicit list the ones we need.  Could you try it out and see if it becomes too unwieldy?

Reply:
That's fine.  It's a fairly harmless extension.